src/server/firefox/ffExecutionContext.ts

```diff
@@ -90,7 +90,7 @@ export class FFExecutionContext implements js.ExecutionContextDelegate {
     await this._session.send('Runtime.disposeObject', {
       executionContextId: this._executionContextId,
       objectId: handle._objectId,
-    }).catch(error => {});
+    });
```

Comment:
Why this change?

Reply:
It's caller's responsibility to catch exceptions if any and it does so now in .dispose().